functions/units/index.js

```diff
@@ -0,0 +1,49 @@
+const functions = require('firebase-functions');
+const admin = require('firebase-admin');
+const dbref = admin.firebase().ref();
```

Comment:
Something tells me this should probably be inside the functions

Reply:
You mean the firebase database reference? What is your reasoning behind that?